src/azure-cli/azure/cli/command_modules/acr/tests/latest/test_acr_commands.py

```diff
@@ -386,7 +386,7 @@ def test_acr_encryption_with_cmk(self, key_vault, resource_group):
 
         # create the registry with CMK encryption enabled using the user-assigned identity
         result = self.cmd('acr create --name {registry_name} --resource-group {rg} --sku premium --identity {identity_id} --key-encryption-key {key_id}', checks=[
-            self.check('identity.type', 'UserAssigned'),
+            self.check('identity.type', 'userAssigned'),
```

Comment:
Why did everything change to the first lower case?

Reply:
As I said above.